packages/logger/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "@apppium/logger",
+  "version": "7.0.1",
+  "author": "https://github.com/appium",
+  "description": "A Universal Logger For The Appium Ecosystem",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/appium/appium.git",
+    "directory": "packages/logger"
+  },
+  "main": "build/index.js",
```

Comment:
it looks like tests are looking for index.js inside of the /lib folder and don't understand the manifest settings: https://github.com/appium/appium/actions/runs/9299443265/job/25593397268#step:5:30

We should either change the manifest and put the index file under /lib or fix the smoke test script